### PR TITLE
Gris 46.add participant state

### PR DESCRIPTION
**Pull request by paulinaborys**
I've modified the create properties function in graph api service so that we don't have to handle every composite property individually.

### CODE REVIEW

File: grisera_api/participant_state/participant_state_model.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# class Glasses(str, Enum):
+#     """
+#     The glasses
+#
+#     Attributes:
+#         yes (str): Glasses
+#         no (str): No glasses
+#     """
+#     yes = "yes"
+#     no = "no"
```
Comment (RioluM) [inline grisera_api/participant_state/participant_state_model.py:32]: I think it can be deleted

**Review by RioluM — COMMENTED** [review]
With addition of participant state, there should go changes in participant model as well, I think.

**Comment by paulinaborys** [comment]
@RioluM what do you mean by changes in the participant model? Should I delete repeating properties?

**Comment by RioluM** [comment]
Yes, according to Contextual Information

File: grisera_api/graph_api_service.py [hunk 2]
```diff
@@ -3,6 +3,60 @@
...
+    Returns:
+        Request body
+    """
+    request_body = []
+    for dict_property in list_of_dicts:
+        keys = list(dict_property.keys())
+        values = list(dict_property.values())
+        for k, v in zip(keys, values):
+            request_body.append({"key": k, "value": v})
+    return request_body
```
Comment (mrwrob) [inline grisera_api/graph_api_service.py:22]: I think using `items()` would make it a little bit more readable:
```
    request_body = []
    for dict_property in list_of_dicts:
        for k, v in dict_property.items():
            request_body.append({"key": k, "value": v})

    return request_body
```
We can go even further using list and dict comprehension combined:
```
    request_body = [{"key": k, "value": v}
                    for dict_property in list_of_dicts for k, v in dict_property.items()]
    return request_body
```
Although I've some doubts if it's still readable :wink:

**Comment by paulinaborys** [comment]
I hope it is readable now 🙏

File: grisera_api/graph_api_service.py [hunk 3]
```diff
@@ -3,6 +3,60 @@
 from pydantic import BaseModel
 
 
+def create_properties_from_list_of_dict(list_of_dicts: list):
+    """
+    Creates request body from list of dictionaries with properties
+
+    Args:
+        list_of_dicts (dict): list of dictionaries of properties
```
[the remark this answers is not in the document]
Comment (paulinaborys) [reply grisera_api/graph_api_service.py]: It's list 👍

**Review by mrwrob — APPROVED** [review]
Not is perfectly readable 🤣

**Pull request merged**